prime-router/src/main/kotlin/history/azure/DeliveryFunction.kt

```diff
@@ -261,6 +267,40 @@ class DeliveryFunction(
         }
     }
 
+    /**
+     * API for searching for submitters for a specific receiver
+     */
+    @FunctionName("getSubmittersV1")
+    fun getSubmitters(
+        @HttpTrigger(
+            name = "getSubmittersV1",
+            methods = [HttpMethod.POST],
+            authLevel = AuthorizationLevel.ANONYMOUS,
```

Comment:
This is if we wanted to hook into the azure authentication service. and could specify the API key that way, but since the auth is all handled by our code it the authLevel for the function `ANONYMOUS`